README.org

```diff
@@ -147,6 +147,58 @@ If you find a working configuration for any other debug adapter please submit a
 
 See [[https://microsoft.github.io/debug-adapter-protocol/implementors/adapters/][microsofts list]] for other adapters, your mileage will vary.
 
+* Performance
+
+Some minor gains to performance in the debugger can be achieved in changing Emacs configuration values for process interaction and garbage collection.
+
+** =gc-cons-threshold=
+
+This variable controls the frequency of garbage collection in Emacs.
+Too high a value will lead to increased system memory pressure and longer stalls, and too low a value will result in extra interruptions and context switches (poor performance).
+
+According to [[https://emacs-lsp.github.io/lsp-mode/page/performance/#adjust-gc-cons-threshold][lsp-doctor]], =100mb= is likely a better setting.
```

Comment:
Might it be better to quote Eli directly:
> Gnu Emacs Maintainer Eli Zaretskii:
> My suggestion is to repeatedly multiply gc-cons-threshold by 2 until you stop seeing significant improvements in responsiveness, and in any case not to increase by a factor larger than 100 or somesuch. If even a 100-fold increase doesn't help, there's some deeper problem with the Lisp code which produces so much garbage, or maybe GC is not the reason for slowdown.

Or for simplicity one could use 
```emacs-lisp
(setq gc-cons-threshold 80000000)
```
which is original value times 100

https://www.reddit.com/r/emacs/comments/brc05y/is_lspmode_too_slow_to_use_for_anyone_else/eofulix/

I think its better to link to the sources directly.